src/io.jl

```diff
@@ -0,0 +1,130 @@
+# Input/ Output of Unfold results
+"""
+    save_results(results::DataFrame, bids_root::String; 
+        save_folder::String="Unfold",
+        overwrite::Bool=false)
+
+Function to save unfold models in your BIDS root folder. Automatically creates a save_folder (default = "Unfold") in the derivatives and subsequentely safes each model in results according to BIDS.
+
+If overwrite is false the function will check for already saved results in the same folder and with the same name i.e. same subject/session/task/run.
+"""
+function save_results(results::DataFrame, bids_root::String;
+    save_folder::String="Unfold",
+    overwrite::Bool=false)
+
+    # Make folder to save in
+    save_in = joinpath(bids_root, "derivatives", save_folder)
```

Comment:
right now there is no option to overwrite the BIDS path and to just safe it wherever (tm)

probably not _that_ important as it is called UnfoldBIDS ;)

Reply:
I think it's a good idea to force the user to save bids compatible.

And anyway, bids_root doesn't have to be a BIDS folder, it _can_ be wherever, you just still will get the path/derivatives/folder/sub... structure